geom/dPoly.cpp

```diff
@@ -995,16 +997,18 @@ void dPoly::reverseMarkedPolys(std::map<int, int> const & mark) {
   return;
 }
 
-std::vector<int> dPoly::get_startingIndices() const{
-	std::vector<int> start_indices;
+const std::vector<int>& dPoly::getStartingIndices() const{
 
+	if ((int) m_startingIndices.size() == m_numPolys +1) return m_startingIndices;
```

Comment:
This logic looks counterintuitive. Maybe a comment can be added.

Reply:
I guess it ensures that m_startingIndicies (whose purpose is unclear) is always in sync with number of polygons?